gradle-plugin/it/src/it/service/expected/openshift.yml

```diff
@@ -133,7 +133,7 @@ items:
     name: svc1
   spec:
     port:
-      targetPort: 8080
```

Comment:
why is value different in this case?

Reply:
> why is value different in this case? I see in this case it's `port1` instead of `http`

because this is the service:
 
```yaml
- apiVersion: v1
  kind: Service
  metadata:
    annotations:
      app.openshift.io/vcs-ref: "@ignore@"
      jkube.eclipse.org/git-url: "@ignore@"
      app.openshift.io/vcs-uri: "@ignore@"
      jkube.eclipse.org/git-commit: "@ignore@"
      jkube.eclipse.org/git-branch: "@ignore@"
    labels:
      app: service
      provider: jkube
      version: "@ignore@"
      group: org.eclipse.jkube.integration.tests.gradle
    name: svc1
  spec:
    ports:
    - name: port1
      nodePort: 301243
      port: 8080
      protocol: TCP
      targetPort: 8080
    selector:
      app: service
      provider: jkube
      group: org.eclipse.jkube.integration.tests.gradle
    type: LoadBalancer
```

the port name is `port1` not `http`